packages/buttons/src/styled/StyledExternalIcon.ts

```diff
@@ -28,6 +28,8 @@ export const StyledExternalIcon = styled(NewWindowIcon).attrs({
   ${props => retrieveComponentStyles(COMPONENT_ID, props)};
 `;
 
+StyledExternalIcon.displayName = 'StyledExternalIcon';
```

Comment:
Do we need to add the `displayName` here? I think it is explicitly the same because of the variable name.
![Screen Shot 2022-11-03 at 12 32 25 PM](https://user-images.githubusercontent.com/12474067/199779361-aad3d982-7bbc-4708-afc3-baf273d43c74.png)

Reply:
The `useText` helper requires a display name as one of the arguments, so I thought that meant I had to have one explicitly declared. I can take a quick look and get back to you, though!

Reply:
Was able to remove it. 👍